corehq/apps/export/models/new.py

```diff
@@ -1087,7 +1093,14 @@ def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=F
                 identifier,
             )
 
-            current_schema.record_update(app.copy_of or app._id, app.version)
+            # Only record the version of builds on the schema. We don't care about
+            # whether or not the schema has seen the current build because that always
+            # gets processed.
+            if app.copy_of:
+                current_schema.record_update(app.copy_of, app.version)
```

Comment:
this is kind of sneaky. we no longer record the app.version of the current app, since it's then easier to figure out how many builds we've gone through. i think this is ok because we always process the most current app